workflow-steps/cache/output/main.js

```diff
@@ -5978,7 +5978,14 @@ function hashKey(key2) {
   const globHashes = globsToHash.map((globPattern) => {
     return hashFileContents(globPattern);
   });
-  return [...hardcodedKeys, ...globHashes].join(" | ");
+  const hashCollections = [...hardcodedKeys, ...globHashes];
+  if (hashCollections.length > 1) {
+    return hash(hashCollections.join(" | "));
+  }
+  return hashCollections.join(" | ");
+}
```

Comment:
nit: formatting lol, and on the post.js file too
```suggestion
}

```

Reply:
it's auto-generated from the build command 😢

Reply:
Oh woops, did not realize. Thought I was still looking at the .ts files.